internal/cli/deployments/connect.go

```diff
@@ -287,12 +289,27 @@ func (opts *ConnectOpts) connectToAtlas() error {
 		return err
 	}
 
+	// Connectionstrings are empty when the server is not in IDLE
+	// r.ConnectionStrings.PrivateSrv == nil and r.ConnectionStrings.StandardSrv == nil
+	if r.StateName == nil || *r.StateName != "IDLE" {
+		return errCouldNotConnectClusterNotIdle
+	}
+
+	// This field is optional, if not set, throw an error
+	if r.ConnectionStrings == nil {
```

Comment:
same as above

Reply:
I actually want to break the flow in case `ConnectionStrings == nil`. If I use the getter I'll get the default value, which is not what I want.

Reply:
there's `GetConnectionStringsOK()` but in this case I don't think it makes it better